pycbc/events/stat.py

```diff
@@ -1481,10 +1486,9 @@ def single(self, trigs):
             # exists if accessed via coinc_findtrigs
             self.curr_tnum = trigs.template_num
         except AttributeError:
-            # exists for SingleDetTriggers
+            # exists for SingleDetTriggers & pycbc_live get_coinc
             self.curr_tnum = trigs['template_id']
-            # Should only be one ifo fit file provided
-            assert len(self.ifos) == 1
```

Comment:
Max is right, this is just removing a block which shouldn't be one anymore

Plus according to [python docs](https://wiki.python.org/moin/UsingAssertionsEffectively) we should really only use assert for debugging our internal logic
```
Assertions should *not* be used to test for failure cases that can occur because of bad user input 
```

Reply:
Yes, `assert` is basically a way to explicitly state what you are assuming at that point.

Reply:
I think we're happy with this then